webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -199,350 +204,593 @@ final case class NodeConfigIdInfo(
+     *    "gar": {                   // global agent run (interval)
+     *      "i": 15,                 // interval
+     *      "sm": 0,                 // start minute
+     *      "sh": 0,                 // strart hour
+     *      "st": 4                  // splay time
+     *    },
+     *    "nar": {                   // node agent run (interval)
+     *      ...                      // same as gar
+     *    },
+     *  },
+     *  "rs": [                       // rules
+     *    {
+     *      "rid": "4bb75daa-a82f-445a-8e8e-af3e99608ffe",      // rule id
+     *      "ds": [                                              // directives
+     *        {
+     *          "did": "73e069ea-de00-4b5d-a00e-012709b7b462",  // directive id
+     *          "cs": [                                         // components
+     *            {                                             // block
+     *              "bid": "my main block",                     // block (component) id
+     *              "rl" : "weighted"                           // reporting logic
+     *              "scs": [                                    // sub components
+     *                 {                                        // component
+     *                    "vid": "Command execution",             // values (component) id
+     *                    "vs": [
+     *                      [],                                 // match value with None/None pair
+     *                      [ "/bin/true" ],                    // match value with same unexpanded/expanded
+     *                      [ "${file}", "/tmp" ],              // match value with different unexpanded/expanded
+     *                      {                                   // report id value
+     *                        "id": "37c57e98-328d-4cd2-8a71-33f2e449ba51",
+     *                        "v": "${file}"
+     *                      }
+     *                    ]
+     *            },
+     *             ...
+     *        ]
+     *      },
+     *      {
+     *        "rid": "hasPolicyServer-root",
+     *        "ds": [
+     *          {
+     *            "did": "common-hasPolicyServer-root",
+     *            "s": true,                                   // system - only mandatory when true
+     *            "cs": [ ...
+     *     "os": [                                             // overrides
+     *       { "p": {                                          // policy
+     *          "rid": "a3a796b9-8499-4e0b-86c5-975fc5a13505",
+     *          "did": "9b0dc972-f4dc-4aaa-bae6-1189cf9074b6"
+     *       },{
+     *         "ob": {                                         // overridden by
+     *           "rid": "2278f76f-28d3-4326-8199-99561dd8c785",
+     *           "did" "093a494b-1073-49e9-bb1e-3c128c7f6a42"
+     *        }
+     *     ]
+     */
+
+    // name are false for startHour / splayHour and no override
```

Comment:
i don't understand this comment

Reply:
ah, I copy/copied it. It's on the version 7 for agentrun